pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -217,6 +225,16 @@ func (p *LaunchTemplateProvider) volumeSize(quantity *resource.Quantity) *int64
 	return aws.Int64(quantity.ScaledValue(resource.Giga))
 }
 
+// Invalidate deletes a launch template from cache if it exists
+func (p *LaunchTemplateProvider) Invalidate(ctx context.Context, ltName string) {
+	p.Lock()
+	defer p.Unlock()
+	defer p.cache.OnEvicted(p.onCacheEvicted)
```

Comment:
Ugh, does Delete() call the OnEvicted handler?

Reply:
yeah :(